src/components/schema-display/schema-display.scss

```diff
@@ -4,4 +4,11 @@
     background: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiPjxwYXRoIGQ9Ik0yMCAyNGgtMjB2LTIyaDNjMS4yMjkgMCAyLjE4LTEuMDg0IDMtMmg4Yy44Mi45MTYgMS43NzEgMiAzIDJoM3Y5aC0ydi03aC00bC0yIDJoLTMuODk4bC0yLjEwMi0yaC00djE4aDE2di01aDJ2N3ptLTEwLTRoLTZ2LTFoNnYxem0wLTJoLTZ2LTFoNnYxem02LTVoOHYyaC04djNsLTUtNCA1LTR2M3ptLTYgM2gtNnYtMWg2djF6bTAtMmgtNnYtMWg2djF6bTAtMmgtNnYtMWg2djF6bTAtMmgtNnYtMWg2djF6bS0xLTdjMCAuNTUyLjQ0OCAxIDEgMXMxLS40NDggMS0xLS40NDgtMS0xLTEtMSAuNDQ4LTEgMXoiLz48L3N2Zz4=');
```

Comment:
Instead of seeing the svg as the background we should use the icon component on the fluent ui component library